include/SFML/System/Angle.inl

```diff
@@ -55,104 +53,104 @@ constexpr Angle::Angle() = default;
 ////////////////////////////////////////////////////////////
 constexpr float Angle::asDegrees() const
 {
-    return m_degrees;
+    return m_radians * (180.f / priv::pi);
```

Comment:
Note that these constructs [already carry a baked in rounding error](https://godbolt.org/z/ThfWPfM1d).

Reply:
Same with calls to `asDegrees()`, I guess it could have a double overload, I'm not sure if we have to care that much? Just pointing it out

Reply:
Are you saying that we could get marginally better accuracy if we used a single literal to represent 180 / pi rather than using this particular expression?

Reply:
`57.295776` - float (Compiler Explorer)  
`57.2957802` - double (Compiler Explorer)  
`57.295779513082323402053960025447` - actual (Windows calculator)

`0.000003513082323402053960025447` - difference float  
`0.000000686917676597946039974553` - difference double

I guess in theory you could also do the full math in doubles and only covert at the end, but I feel like if you care about such a precision, you may want to implement the mathy bits yourself.

The change makes sense to me, but could also be a separate PR.